api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupDataQualityReport.java

```diff
@@ -250,14 +266,14 @@ public String getVersion() {
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Data_Quality_Report.xls",
+              "Data_Quality_Report_v2.xls",
```

Comment:
@KelvinWanyama did we need to rename the report to v2? 

@psmatsinhe

Reply:
Generally we manage the version the partners receive via the module version, which usually makes it uncessary to track version in the report name. 

Another thing is that we haven't done this for the other reports, so seems strange to be treating this report differently. What do you think?

Reply:
@nthfloor  Pinki instructed that we rename the report to v2 and it is the reason why I made the updates here. Could probably revoke if otherwise. Probably we can clear with Pinki then make changes accordingly.

Reply:
Ok I'll follow up with her about this one. thanks